Kudu.FunctionalTests/Jobs/WebJobsTests.cs

```diff
@@ -12,12 +12,15 @@
 using Kudu.Client.Infrastructure;
```

Comment:
Need lots more tests. Perhaps some/most of these should be unit tests. Tests for:

- TriggeredJobSchedule class and all the logic in Reschedule
- TriggeredJobsScheduler class (lots going on in there)

Your end to end functional shows a happy path, but I want to make sure we have test coverage for all the rest.